src/tests/unit/onbase-dao.test.js

```diff
@@ -0,0 +1,128 @@
+const chai = require('chai');
```

Comment:
We should start using ES6 style to import module: https://github.com/osu-mist/terms-api/blob/master/src/tests/unit/terms-serializer.test.js#L1-L7